rollup.config.ts

```diff
@@ -96,6 +94,10 @@ const buildNode = {
       extensions: ['.js', '.jsx', '.ts', '.tsx'],
     }),
     typescript({
+      /**
+       * @fixme Export only "msw/node" relevant type declarations
+       * to enable auto-imports from "msw/node".
+       */
       useTsconfigDeclarationDir: true,
```

Comment:
If this is set to `false` the entire library declarations will be emitted to `node/`:

```
node/
  handlers/
    RestHandler.d.ts
  context/
   json.d.ts
  ...
  index.d.ts
```

This isn't correct. Only the node-related declarations (those in `src/node`) must be emitted to the `node/` directory.